zephyr-apps/mppt/prj.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_GPIO=y
+CONFIG_STDOUT_CONSOLE=y
```

Comment:
Not sure which of these configs are for logging, but for my timers app, I only needed `CONFIG_LOG=y` to use `LOG_INF`

Reply:
CONFIG_GPIO is for enabling GPIO which I use in the app. 

CONFIG_STDOUT_CONSOLE
```
This option directs standard output (e.g. printf) to the console
device, rather than suppressing it entirely. See also EARLY_CONSOLE
option.
```

ie. provides more verbose logs